### PR TITLE
Symfony 4.0

### PR DESCRIPTION
**Pull request by lsmith77**
based on #151

for php 7.2 we need https://github.com/zendframework/ZendDiagnostics/pull/83

### CODE REVIEW

File: Tests/app/services.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+services:
```
Comment (lsmith77) [inline Tests/app/services.yml]: this is a "hack" to expose the private service for the PathHelperTest

**Comment by lsmith77** [comment]
I guess then we shall tag 2.6.0 ? 

https://github.com/liip/LiipMonitorBundle/compare/2.5.2...master

**Comment by kbond** [comment]
Sure, I can tag.

**Comment by kbond** [comment]
Done: https://github.com/liip/LiipMonitorBundle/releases/tag/2.6.0

**Comment by lsmith77** [comment]
gracias!

**Pull request merged**